doc/architecture/wagons/README.md

```diff
@@ -262,11 +262,17 @@ If wished, the attributes can be included in the CSV-File that is generated when
 
 ### Make attributes searchable
```

Comment:
This isn't really a wagon documentation, I would maybe move this somewhere else, and only explain how to add something to SEARCHABLE_ATTRS in wagons here :rocket: